orangecontrib/text/topics/topics.py

```diff
@@ -57,6 +58,24 @@ def get_value(self, model, *args, **kwargs):
         return self.epochs / model.passes
 
 
+def infer_ngrams_corpus(corpus):
```

Comment:
I am considering adding a `lru_cache` decorator to this function since infer_ngrams_coprus is called on several locations for the same corpus. Before, Sparse2Corpus was cached in the corpus; now, lru_cache can be used. It will return cached results only for completely the same corpus. What do you think?

```suggestion
@lru_cache
def infer_ngrams_corpus(corpus):
```